apps/api/src/domain/domainModelValidators/tagValidator.ts

```diff
@@ -20,11 +21,15 @@ const tagValidator: DomainModelValidator = (
 
   const innerErrors: InternalError[] = [];
 
-  const { text, id } = dto as PartialDTO<Tag>;
+  const { text, id, published } = dto as PartialDTO<Tag>;
 
   if (!isStringWithNonzeroLength(text))
     innerErrors.push(new TagHasNoTextError(id));
 
+  // TODO Validate inherited properties on the base class
```

Comment:
Do this now

Reply:
On second thought, I'm going to wait. There's an important design decision to be made here and I don't want to go there prematurely.